Formula/sratoolkit.rb

```diff
@@ -44,24 +43,32 @@ def install
     ncbi_vdb_build = buildpath/"ncbi-vdb-build"
     ncbi_vdb_source.install resource("ncbi-vdb")
     cd ncbi_vdb_source do
-      system "./configure",
-        "--prefix=#{buildpath/"ncbi-vdb-prefix"}",
-        "--with-ngs-sdk-prefix=#{ngs_sdk_prefix}",
-        "--build=#{ncbi_vdb_build}"
+      # Fix error: `pshufb' is not supported on `generic64.aes'
+      # Upstream ref: https://github.com/ncbi/ncbi-vdb/issues/14
+      on_linux { inreplace "libs/krypto/Makefile", "-Wa,-march=generic64+aes", "" }
+
+      system "./configure", "--prefix=#{buildpath/"ncbi-vdb-prefix"}",
+                            "--with-ngs-sdk-prefix=#{ngs_sdk_prefix}",
+                            "--build=#{ncbi_vdb_build}"
       ENV.deparallelize { system "make" }
     end
 
     # Fix the error: ld: library not found for -lmagic-static
     # Upstream PR: https://github.com/ncbi/sra-tools/pull/105
     inreplace "tools/copycat/Makefile", "-smagic-static", "-smagic"
 
-    system "./configure",
-      "--prefix=#{prefix}",
-      "--with-ngs-sdk-prefix=#{ngs_sdk_prefix}",
-      "--with-ncbi-vdb-sources=#{ncbi_vdb_source}",
-      "--with-ncbi-vdb-build=#{ncbi_vdb_build}",
-      "--build=#{buildpath}/sra-tools-build"
+    # Fix the error: utf8proc.o: linker input file unused because linking not done
+    # Upstream issue: https://github.com/ncbi/sra-tools/issues/283
+    on_linux do
+      inreplace "tools/driver-tool/utf8proc/Makefile",
+                "$(CC) $(LDFLAGS) -shared", "#{ENV.cc} $(LDFLAGS) -shared"
```

Comment:
Not sure about best way to fix this. `$(CC) == "gcc -c"`, which causes issues when building shared library.

For now, just replaced with `ENV.cc`.

Reply:
Looks like this is building its own `utf8proc`. Can we use Homebrew `utf8proc` instead?

Reply:
I wasn't able to find a way when I skimmed through. 
One of previous CI runs had `utf8proc` as dependency, but it didn't do anything.

On side note, the build system is very strange here.
When I tried building from source on macOS, it didn't pick up any libs for me:
```
checking for Python 3... /usr/bin/python3
checking for fuse library... no
checking for hdf5 library... no
checking for magic library... no
checking for xml2 library... no
```

Reply:
Their `configure` script has options that might help:
```
Optional Packages:
  --with-fuse-prefix=DIR    search for fuse files in DIR
  --with-hdf5-prefix=DIR    search for hdf5 files in DIR
  --with-magic-prefix=DIR    search for magic files in DIR
  --with-xml2-prefix=DIR    search for xml2 files in DIR
```
The `configure` files of the resources seem to have the same options.

Reply:
I actually tried configuring `ncbi-vdb` resource with options to `hdf5` and `magic`. It helped find `libmagic`, but `hdf5` wasn't found.

From Linux CI logs, I recall a similar situation where only `libmagic` was found.

The `configure` script ends up calling a Perl script, so it isn't a typical build system.

---

I just tried installing current Intel Big Sur bottle and the only dynamic links are:
- /usr/lib/libSystem.B.dylib
- /usr/lib/libc++.1.dylib

Downloading the Linux bottle, the links are:
- ld-linux-x86-64.so.2
- libc.so.6
- libdl.so.2
- libm.so.6
- libpthread.so.0
- libstdc++.so.6

Reply:
Looks like Linux may also have issue finding `hdf5` from local install attempt:
```
checking for fuse library... no
checking for hdf5 library... no
checking for magic library... yes
checking for xml2 library... yes
```
May need to try similar work around to macOS.